### PR TITLE
[Group 1] Add Elevator Commands

### PR DESCRIPTION
**Pull request by vincentw921**

### CODE REVIEW

File: Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorClimbCommand.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.stuypulse.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Elevator;
+
```
Comment (vincentw921) [inline Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorClimbCommand.java:6]: Author tags

File: Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorPickupCommand.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.stuypulse.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Elevator;
+
```
Comment (vincentw921) [inline Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorPickupCommand.java:6]: Author tags

File: Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToScaleCommand.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.stuypulse.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Elevator;
+
+public class ElevatorToScaleCommand extends CommandBase {
```
Comment (vincentw921) [inline Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToScaleCommand.java:11]: Changing Name????

File: Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToScaleCommand.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package com.stuypulse.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Elevator;
+
```
Comment (vincentw921) [inline Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToScaleCommand.java:6]: Author tags

File: Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToSwitchCommand.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package com.stuypulse.robot.commands.elevator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import com.stuypulse.robot.subsystems.Elevator;
+
```
Comment (vincentw921) [inline Group1/src/main/java/com/stuypulse/robot/commands/elevator/ElevatorToSwitchCommand.java:6]: Author tags

**Comment by selym3** [comment]
make a elevator set position command and then just have 4 children for each position u want 
(also have the elvator subsystem person remove the 4 methods for each set height and just have setposition command call .setPosition())

**Comment by selym3** [comment]
Also big thing... CommandBase or InstantCommand =) (make sure to test everything with commands and button bindings too)

**Pull request merged**